plugins/catalog-backend-module-incremental-ingestion/src/router/routes.ts

```diff
@@ -50,13 +49,13 @@ export class IncrementalProviderRouter {
     });
 
     // Clean up and pause all providers
-    router.post(PROVIDER_CLEANUP, async (_, res) => {
+    router.post('/incremental/health', async (_, res) => {
```

Comment:
```suggestion
    router.post('/incremental/cleanup', async (_, res) => {
```

Reply:
Oops. I had a commit in my local that I forgot to push. Good catch.